tests/cmd/stability/main.go

```diff
@@ -236,6 +236,8 @@ func run() {
 		time.Sleep(3 * time.Minute)
 		fta.StartNodeOrDie(physicalNode, node)
 		oa.EmitEvent(nil, fmt.Sprintf("StartNode: %s on %s", node, physicalNode))
+		oa.WaitPodOnNodeReadyOrDie(deployedClusters, node)
+		time.Sleep(3 * time.Minute)
```

Comment:
why sleeping 3 minutes?

Reply:
if the problem is `CheckRecoverOrDie` may fail unexpectedly when pods are not ready. after we have waited for the pods to be ready, IMO, this sleep is not necessary.

Reply:
updated and removed.